openstack/objectstorage/v1/objects/requests.go

```diff
@@ -576,15 +576,25 @@ func CreateTempURL(c *gophercloud.ServiceClient, containerName, objectName strin
 
 // BulkDelete is a function that bulk deletes objects.
```

Comment:
Unrelated to this change, but I wonder if we should mention the (configurable) limit of 10,000 objects per request?

Reply:
I wasn't able to find a reference to a limit [in the API docs](https://docs.openstack.org/api-ref/object-store/#delete-object). Did you find that in Swift or Ceph?

Reply:
I think it's defined by the user in Swift config:

https://github.com/openstack/swift/blob/6d3d4197151f44bf28b51257c1a4c5d33411dcae/etc/proxy-server.conf-sample#L1029-L1034
https://github.com/openstack/swift/blob/e8cecf7fcc1630ee83b08f9a73e1e59c07f8d372/swift/common/middleware/bulk.py#L309

Reply:
wonderful, thank you @stephenfin @EmilienM. I have added that information in the function docs. PTAL